lib/controllers/combined_files.js

```diff
@@ -107,7 +107,7 @@ BT.CombinedFilesController = SC.ArrayController.extend({
 
     files.forEach(function (f) {
       ret.push("/* begin of file %@ */".fmt(f.get('path')));
-      ret.push(f.get('content') || "");
+      ret.push((f.get('content') || "") + ';');
```

Comment:
Isn't this going to be problematic when files turn out to be empty? I would think that a check like 
```
var c = (f.get('content') || '').trim();
if (c && c.lastIndexOf(';') !== c.length - 1) c += ';';
ret.push(c);
```
would solve the problem a bit more reliable. Thoughts?